CMakeLists.txt

```diff
@@ -352,7 +357,7 @@ else ()
   set (_configure_date "Unknown")
 endif()
 include_directories (${CMAKE_BINARY_DIR})
-configure_file (build/version_string.ver.in version_string.ver @ONLY)
+configure_file (${CMAKE_CURRENT_LIST_DIR}/build/version_string.ver.in version_string.ver @ONLY)
```

Comment:
This will not clutter the input:
This is just being explicit on the input value 
configure_file(<input> <output>

The above code does not change anything.  I'll revert to use the implicit pathing like it used to be.